changelog/fragments/update-k8s-controller-runtime-version.yaml

```
@@ -0,0 +1,19 @@
+# entries is a list of entries to include in
+# release notes and/or the migration guide
+
+entries:
+  - description: >
+      Update k8s dependencies in Operator SDK to 0.19.4 and controller-runtime to v0.7.0-alpha.7.
+    kind: "change"
+
+  - description: >
+      For ansible/helm based operators, updated k8s dependencies from v0.18+ to v0.19.4
+    kind: "change"
+
+  - description: >
+      For ansible/helm based operators, updated controller-runtime dependencies to v0.7.0-alpha.7.
+    kind: "change"
+
+  - description: >
+      For helm based operators, updated Helm dependency to v3.4.1 from v3.3.4.
```

Helm-based operators aren't affected, just the `helm-operator` binary. These are different things
```suggestion
      Updated helm-operator's Helm dependency to v3.4.1.
```